types/network/Network.d.ts

```diff
@@ -0,0 +1,1049 @@
+// Type definitions for vis.js 4.21
```

Comment:
Here the version is no longer correct and could be removed.

Also i'm not sure if we should just copy this file into this repository!? But what are the alternatives?
- We could cherry-pick the network commits into this repository. This ways we would keep the git history of the file, but this would be really messy...( Maybe I'll try it! It's a challange! )
- We could just fork the `DefinitelyTyped` repository and maintain the types there. But this would be hard to keep up to date. I like the idea to have the types inside the repository...